apps/fluent-tester/src/E2E/common/NativeTesting/pages/NativeTestingPageObject.win.ts

```diff
@@ -0,0 +1,45 @@
+import { BasePage, By } from '../../../common/BasePage.win';
+
+class NativeTestingPageObject extends BasePage {
+  waitForScrollViewDisplayed(timeout?: number): void {
+    browser.waitUntil(
+      () => {
+        return this.doesScrollViewParentExist();
+      },
+      {
+        timeout: timeout ?? this.waitForPageTimeout,
+        timeoutMsg:
+          'The ScrollView containing all test page buttons was not found. Did you remove the ScrollView or change the testID? The testID must stay constant to align with our native code.',
```

Comment:
we should likely make this common in time and not win32-only but regardless -- since we're codifying a policy rule I'd remove the reference to native code and more ambiguously assert something like `the testID is fixed for testing purposes` since the source of the policy isn't *especially relevant

Reply:
Each Platform's native code will be different, so I believe we'll probably need platform-specific files just like this one for Mac, ios, etc.

Also, I changed the error message to something more ambiguous.

Reply:
might be desktop vs mobile but yep :)